es73x/src/main/scala/tech/beshu/ror/es/ssl/SSLNetty4InternodeServerTransport.scala

```diff
@@ -105,7 +107,10 @@ class SSLNetty4InternodeServerTransport(settings: Settings,
             new ByteArrayInputStream(privateKey.getBytes(StandardCharsets.UTF_8)),
             null
           )
-          if (ssl.clientAuthenticationEnabled) sslCtxBuilder.clientAuth(ClientAuth.REQUIRE)
+          if (ssl.clientAuthenticationEnabled) {
```

Comment:
I noticed we can enable client authentication for internode ssl (mutual auth), but...we don't create keystore on client side  `getClientChannelInitializer` so client can not use its cert. So it looks like it doesn't work correctly.

And I'm not sure if this option is really necessary. Our goal is certs verification for all nodes in cluster, so I think verification performed by one side (client) is sufficient as each node (server) would be verified by other nodes (clients).  What do you think @coutoPL @sscarduzio?

Reply:
agree - one way verification in internode SSL seems to be enough